app/router.js

```diff
@@ -29,9 +38,35 @@ module.exports = app => {
       return result
     })
   })
+
+  app.use(offenderRoute, getOffenderDetails)
+  app.get(`${offenderRoute}/plans`, (req, res) => sentencePlanSummary(req, res))
+
+  // sentence plans summary
+  app.get([offenderRoute, `${offenderRoute}/plans`], sentencePlanSummary)
+
+  // diversity
+  app.get([`${offenderRoute}/edit-plan/:planid(\\d{1,})/diversity`], getDiversity)
```

Comment:
Worth using the numericId const?
app.get([`${offenderRoute}/edit-plan/:planid(numericId)/diversity`]

Reply:
Actually no, just use the edit-plan route :-D

app.get([${editPlanRoute}/diversity]